.github/workflows/conventional-pr.yml

```diff
@@ -0,0 +1,39 @@
+name: Conventional PR
+
+# Controls when the action will run. 
+on:
+  pull_request:
+    branches:
+    - master
+    - develop
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: semantic-pull-request
+        # You may pin to the exact commit or the version.
+        # uses: amannn/action-semantic-pull-request@db6e259b93f286e3416eef27aaae88935d16cf2e
+        uses: amannn/action-semantic-pull-request@v3.4.0
```

Comment:
should we rely on this external repository or fork it under Unity-Technologies (and keep it up-to-date with the original repo when needed)? Personally, I'd suggest the latter.

Reply:
Would you settle for a specific commit hash instead of a fork? You can't reference actions from private repos without a bunch of partial workarounds, and getting approval for another public repo is a very slow and involved process.

Reply:
ugh sure, my point was "what if repo goes away?" but I think I'm fine with this too (this-is-fine emoji here)

Reply:
Referencing a specific commit hash now. Also added an (intentionally vague) comment about having a mirror of the repo on Unity github.